mailbox/tools/indexer/src/test/java/org/apache/mailbox/tools/indexer/SingleMailboxReindexingTaskTest.java

```diff
@@ -0,0 +1,21 @@
+package org.apache.mailbox.tools.indexer;
```

Comment:
Apache license missing

Reply:
Forget it... The file has been removed? The fact that github shows this commit being made 20h ago against 2 days for the others is fooling...